Examples.fshx

```diff
@@ -38,11 +50,79 @@ InstanceOf: CancerPatient
 * communication.language = urn:ietf:bcp:47#en-US "English (Region=United States)"
 * communication.language.text = "English"
 
+
+Instance: mCODEPatientExample02
+InstanceOf: CancerPatient
+* meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/CancerPatient"
+* extension[us-core-race].extension[ombCategory].valueCoding = urn:oid:2.16.840.1.113883.6.238#2186-5 "Not Hispanic or Latino"
+* extension[us-core-race].extension[text].valueString = "Not Latino"
+* extension[us-core-birthsex].valueCode.system = "http://terminology.hl7.org/CodeSystem/v3-AdministrativeGender"
+* extension[us-core-birthsex].valueCode.code = #F 
+* identifier.use = #usual
+* identifier.type = http://terminology.hl7.org/CodeSystem/v2-0203#MR "Medical Record Number"
+* identifier.system = "http://hospital.example.org"
+* identifier.value = "m456"
+* name.family = "Anyperson"
+* name.given[0] = "Eve"
+* name.given[1] = "A."
+* contact.telecom[0].system = #phone
+* contact.telecom[0].value = "999-999-9999"
+* contact.telecom[0].use = #home 
+* contact.telecom[1].system = #email
+* contact.telecom[1].value = "eve.anyperson@example.com"
+* gender = #female
+* birthDate = "1955-05-20"
+* address.line = "456 Smith Lane"
+* address.city = "Anytown"
+* address.state = "MA"
+* address.postalCode = "12345"
+* address.country = "US"
+* communication.language = urn:ietf:bcp:47#en-US "English (Region=United States)"
+* communication.language.text = "English"
+
+
+Instance: mCODEOrganizationExample01
+InstanceOf: Organization
+* identifier.system = "http://hl7.org.fhir/sid/us-npi"
+* identifier.value = "1265714091"
+* active = #true 
+* extension[Status].valueCodeableConcept = SCT#55561003 "Active"
+* name = "Foundation Medicine"
+* contact.telecom.system = #phone
+* contact.telecom.value = "617-418-2200"
+* address.line = "150 Second Street"
+* address.city = "Cambridge"
+* address.state = "MA"
+* address.postalCode = "02141"
+* address.country = "USA"
+
+
+Instance: mCODEPractitionerExample01
+InstanceOf: Practitioner
+* identifier.system = "http://hl7.org.fhir/sid/us-npi"
+* identifier.value = "9988776655"
+* name.family = "Anydoc"
+* name.given = "Kyle"
+* name.prefix = "Dr."
+* gender = #male
+* address.use = "home"
+* address.line = "567 Healthcare Drive"
+* address.city = "Anytown"
+* address.state = "MA"
+* address.postalCode = "12345"
+* address.country = "US"
+* qualification.system = "http://terminology.hl7.org/CodeSystem/v2-0360|2.7"
+* qualification.code = "MD" 
+* qualification.display = "Doctor of Medicine"
```

Comment:
There is no "qualification.display"... so it all gets packed into the qualification.code line. Note that qualification.code is a CodeableConcept.